src/script/components/report-card.ts

```diff
@@ -337,6 +339,20 @@ export class ReportCard extends LitElement {
     this.dispatchEvent(event);
   }
 
+  decideWhereToGo() {
+    const baseOrPublishIffy = baseOrPublish();
+
+    if (baseOrPublishIffy === 'base') {
+      Router.go('/basepackage');
+    }
+    else if (baseOrPublishIffy === 'publish') {
+      Router.go(`/publish?site=${this.currentURL}`);
+    }
+    else {
+      Router.go('/basepackage');
+    }
```

Comment:
Nit: the else statements on new lines seems non-standard JS to me. I expect something like:

```js
if (...) {
    ...
} else if (...) {
   ...
} else {
   ...
}
```

Is this aligned with our linter?

Reply:
Yep! This has been run through our linter. I just ran it again to double check and it has not changed. Funny enough, this is also how ive always done my if / else if statements lol